### PR TITLE
S2U-10 5.1.1.6 Tests & Quizzes: Ability to preview calculated questions in question pools

### PR DESCRIPTION
**Pull request by jesusmmp**
https://sakaiproject.atlassian.net/browse/S2U-10

- Adding incorrect feedback on the preview
- Fixing some regression caused by: https://github.com/sakaiproject/sakai/pull/6139

Result:

image

### CODE REVIEW

**Comment by bgarciaentornos** [comment]
It seems like this PR is incorporating previous commits? Also it appears linked to the wrong jira.

File: samigo/samigo-app/src/webapp/jsf/questionpool/solution.jsp [hunk 1]
```diff
@@ -33,6 +33,15 @@
 	<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <head><%=request.getAttribute("html.head")%>
 <title><h:outputText value="#{questionPoolMessages.sol_q}" /></title>
+
+<script>
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/jsf/questionpool/solution.jsp:37]: @jesusmmp

Could we implement this avoiding jQuery? jQuery could be removed in the future and I prefer using vanilla JS.

**Review by mpellicer — CHANGES_REQUESTED** [review]
It looks great @jesusmmp thank you very much.

Just one comment about the JS snippet.

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Awesome!

**Pull request merged**